src/builder.rs

```diff
@@ -9,6 +9,8 @@ pub struct Builder {
 
     config: Config,
 
+    network: IpNetwork,
```

Comment:
How about the following?

```suggestion
    interface: IpInterface
```